### DISCUSSION TITLE
Rallly now has a dedicated self-hosting repo

### DISCUSSION THREAD

**Discussion by lukevella** [lukevella/rallly#396] [Announcements]
Hi everyone,

Happy to announce that there is a new repo dedicated to self-hosting Rallly. The docker-compose specification has been moved to this repo and there are also more instructions to help get you started. I'll look to improve the experience of self-hosting Rallly in the future but this is just a first step towards that goal.

If anything is unclear or you have ideas on how to make things better, you can ask on this thread or open issues in that repo.

[Go to self-hosting repo &rarr;](https://github.com/lukevella/rallly-selfhosted)

Thank you!

**Comment by joho1968** [comment]
Great initiative! One thing that comes to mind, in particular since it's a Docker environment, is what customizations can be made, and how.

**Reply by lukevella** [reply]
Well at the moment, you are limited to the environmental variables that are mentioned in the readme. In the future, there can be more environment variables added to customize things like the color theme, name of the app etc…  

There's also the option of building a web UI for administrators but that would be a significant effort and I simply do not have the bandwidth to take that on right now.
